libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -299,14 +301,27 @@ ASTPointer<ContractDefinition> ASTJsonImporter::createContractDefinition(Json::V
 	);
 }
 
+ASTPointer<IdentifierPath> ASTJsonImporter::createIdentifierPath(Json::Value const& _node)
+{
+	astAssert(_node["name"].isString(), "Expected 'name' to be a string!");
+
+	vector<ASTString> namePath;
+	vector<string> strs;
+	string nameString = member(_node, "name").asString();
+	boost::algorithm::split(strs, nameString, boost::is_any_of("."));
+	for (string s: strs)
+		namePath.emplace_back(s);
```

Comment:
Could add a nonempty check and a check for all elements and that `strs` is at least one element long.

Reply:
I added a check for non-empty elements and for >=1 elements in total, but I am not sure what you mean with "a check for all elements"

Reply:
a check that all elements are non-empty, I think you did that.

Reply:
Sorry, was a typo.